### PR TITLE
OLS-626: Script to upload CSV files into Google sheet

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Script to upload CSV files into Google sheet

## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [x] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents

- Related Issue #[OLS-626](https://issues.redhat.com//browse/OLS-626)
- Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: scripts/data_collection/README.md [hunk 1]
```diff
@@ -117,3 +117,30 @@ An example:
 python delete_ols_archive.py archives/compressed/b6/b6cf044a-4870-4a8c-a849-1b0016cc8171/202404/25/185345.tar.gz
 ```
 
+## `update_google_sheet.py`
+
+### Description
+
+Script to update Google sheet containing with user feedback and conversation
```
Comment (onmete) [inline scripts/data_collection/README.md:124]: extra `with`

File: scripts/data_collection/update_google_sheet.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
+"""Script to update Google sheet containing with user feedback and conversation history.
+
+In order to use this script, file with client secret to Google API must be present (symlinked etc.)
```
Comment (onmete) [inline scripts/data_collection/update_google_sheet.py:3]: Would be useful to say what exactly is needed.

File: scripts/data_collection/README.md (hunk 1 above)
Comment (tisnik) [reply scripts/data_collection/README.md]: fixed @onmete

File: scripts/data_collection/update_google_sheet.py (hunk 2 above)
Comment (tisnik) [reply scripts/data_collection/update_google_sheet.py:3]: fixed @onmete

**Pull request merged**